topics/assembly/tutorials/vgp_workflow_training/workflows/main_workflow.ga

```diff
@@ -1,3635 +0,0 @@
-{
```

Comment:
yes. But let them stabilize a little.They are tested via IWC mechanics. It is in fact quire difficult to test assembly workflows because underlying test data is usually large.  Let's keep them outside for now.

Reply:
We just need a workflow here that reflects the steps that are in the tutorial, as a resource for learners and instructors. If the tutorial corresponds exactly to the IWC workflow, just add a copy here. If not, please generate a workflow that matches the tutorial. If tests are problematic we will accept it without. 

I can help if needed.

Reply:
Can you guarantee that users will get the workflow via TRS and not through the potentially out of date and not automatically updatable workflow in the GTN ?

Reply:
The GTN workflow only needs to be up-to-date with the hands-on steps as written in the tutorial, not the latest version of the VGP workflow in general. In the tutorial text you can still point to the WFH/IWC as the main source for the latest version.

 (Maybe I misunderstood what you mean?)

Reply:
fwiw, in this version of the tutorial learners are not instructed to use a WF at all, only perform all the steps manually.